plutus-tx/test/Plugin/primitives/int2.plc.golden

```diff
@@ -1 +1 @@
-(program (con 8 ! 1))
\ No newline at end of file
+(program (con 1))
```

Comment:
Same as `int.plc.golden`.

Reply:
Again, I'd prefer to tidy these up later.